docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -0,0 +1,55 @@
+.. post:: Apr 29, 2020
+   :tags: portland-2020, prague-2020, conferences, covid-19
+
+Write the Docs 2020 conferences going virtual
+==================================
+
+Hello everyone,
+
+We hope that you're holding up during these trying times.
+We are all stuck at home, but still thinking about our docs community :)
+
+Due to the global events and travel restrictions, which are not expected to relax in the near future, we concluded that **we must cancel our on-site Portland and Prague conferences this year.**
+Even if the official travel and event restrictions will be relaxed by the scheduled dates, the safety of the community and everyone else comes first, so **we are converting the conferences to online-only virtual events with an adjusted format.**
+
+The Portland virtual conference will be held in August, exact dates will be announced soon. The Prague virtual conference will be held in the Autumn, exact dates and Call for Proposals announcement will be announced in early summer.```
+Your Portland conference ticket
+--------------------------------
+
+If you currently have a ticket to Write the Docs Portland, your options are:
+
+- Pre-order a ticket to the online event for $100 and get the rest of your original ticket amount refunded.
+- Pre-order a ticket to the online event for $100 and contribute the rest of your original ticket amount to the Write the Docs organization. We are a self-sustaining community and our events are our main source of income, so your contributions help ensure that we can continue to organize events and maintain the global community in years to come.
+- Get a full refund of your original ticket amount on **May 10th, 2020**. This option will happen automatically if you don't select either of the previous options.
```

Comment:
Isn't this term a bit short? I think we should allow two weeks from sending this out.

Reply:
If we change this, change ticket-choices.rst too.

Reply:
+1 to extending the deadline,  least 2 weeks if not 3?

Reply:
Yea, I'm happy to extend, I just put an initial date.